File: 백준/Bronze/2753. 윤년/윤년.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        int res = 0;
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        if((num % 4 == 0 && num % 100 != 0)||(num % 400 == 0))
            res = 1;
        System.out.println(res);
    }
}

File: 백준/Bronze/2753. 윤년/윤년.kt
import java.util.Scanner;
fun main(){
    val sc = Scanner(System.`in`)
    var y = sc.nextInt()
    var res = if((y % 4 == 0 && y % 100 != 0)||(y % 400 == 0))
    1
    else
    0
    print(res)
}